### PR TITLE
Update SimplifiedCharts.md to Unicode 15.0

### PR DESCRIPTION
**Pull request by graphemecluster**

### CODE REVIEW

File: SimplifiedCharts.md [hunk 1]
```diff
@@ -480,6 +480,12 @@ U+8F9A	kRSUnicode	159'.12
 ````
 
 
+### 辶
+````
+U+8FB6	kRSUnicode	162'.0
```
Comment (hfhchan) [inline SimplifiedCharts.md:485]: ?

**Review by hfhchan — CHANGES_REQUESTED** [review]

File: SimplifiedCharts.md (hunk 1 above)
Comment (graphemecluster) [reply SimplifiedCharts.md:485]: Why what I generated is different from you – did you remove them explicitly before?

Comment (graphemecluster) [reply SimplifiedCharts.md:485]: Should 黄 also be removed? If so, how about 丬 (`90.0 90'.0`)?

Comment (hfhchan-plb) [reply SimplifiedCharts.md:485]: >  You generated them by writing a script right?

I think I did them manually LOL

`U+8FB6` should be corrected in newer version of Unihan.

I think we can keep 黄, same case as 丬 basically.

Comment (graphemecluster) [reply SimplifiedCharts.md]: Did you use the wrong account again 😂

Comment (hfhchan) [reply SimplifiedCharts.md]: > You generated them by writing a script right?

I think I did them manually LOL

U+8FB6 should be corrected in newer version of Unihan.

I think we can keep 黄, same case as 丬 basically.

**Pull request merged**